README.md

```diff
@@ -15,9 +15,9 @@ The Conversation connector consists of a set of Node.js functions deployed as Cl
 
 This diagram illustrates the default configuration of a Conversation connector deployment.
 
-![Default Pipeline Architecture](readme_images/connector_pipeline.png)
+![Default Pipeline Architecture](readme_images/conv-connector-arch-with-multipost.jpg)
```

Comment:
I'm wondering if we leave the original in this main repo and then add the updated one to the facebook readme? Right now we only support multipost for facebook.  Even then, I wonder if we should get someone from design to cleanup the picture. I had just done this quickly in powerpoint over the png and the arrows and color don't really match the original.

Reply:
I think it's OK to have the diagram here (although we do need it redone by a designer). It already shows `batched-messages`, which is also Facebook-specific.